x-pack/test/functional/fixtures/kbn_archiver/canvas/reports.json

```diff
@@ -0,0 +1,79 @@
+{
+  "attributes": {
+    "@created": "2020-10-21T23:50:26.466Z",
+    "@timestamp": "2020-10-22T23:44:48.387Z",
+    "assets": {},
+    "colors": [
+      "#37988d",
+      "#c19628",
+      "#b83c6f",
+      "#3f9939",
+      "#1785b0",
+      "#ca5f35",
+      "#45bdb0",
+      "#f2bc33",
+      "#e74b8b",
+      "#4fbf48",
+      "#1ea6dc",
+      "#fd7643",
+      "#72cec3",
+      "#f5cc5d",
+      "#ec77a8",
+      "#7acf74",
+      "#4cbce4",
+      "#fd986f",
+      "#a1ded7",
+      "#f8dd91",
+      "#f2a4c5",
+      "#a6dfa2",
+      "#86d2ed",
+      "#fdba9f",
+      "#000000",
+      "#444444",
+      "#777777",
+      "#BBBBBB",
+      "#FFFFFF",
+      "rgba(255,255,255,0)"
+    ],
+    "css": ".canvasPage {\n\n}",
+    "height": 8,
+    "isWriteable": true,
+    "name": "The Very Cool Workpad for PDF Tests",
+    "page": 0,
+    "pages": [
+      {
+        "elements": [
+          {
+            "expression": "shape \"square\" fill=\"#4cbce4\" border=\"rgba(255,255,255,0)\" borderWidth=0 maintainAspect=false | render",
+            "id": "element-e05237c6-0fda-49e7-84fc-b8573739b515",
+            "position": {
+              "angle": 0,
+              "height": 18,
+              "left": -1,
+              "parent": null,
+              "top": -1,
+              "width": 15.5
+            }
+          }
+        ],
+        "groups": [],
+        "id": "page-c0e601aa-1c72-4d4a-b73f-8fb72a9d8d3a",
+        "style": {
+          "background": "#FFF"
+        },
+        "transition": {}
+      }
+    ],
+    "variables": [],
+    "width": 8
+  },
+  "coreMigrationVersion": "8.0.0",
```

Comment:
Another 8.0 export.

Reply:
Does that mean simply changing the value to 7.x, or rebasing off of the 7.x branch, or something else?

Reply:
https://github.com/elastic/kibana/pull/102533#discussion_r673150222

Reply:
Done, merging soon.